addons/t9n/views/t9n_resource_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="t9n.resource_view_form" model="ir.ui.view">
+        <field name="name">Resource</field>
+        <field name="model">t9n.resource</field>
+        <field name="arch" type="xml">
+            <form string="Resource Form">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="file_name" readonly="True"/>
```

Comment:
We probably always want the `file_name` to be readonly, right? If yes, then you can add this as a field parameter and it will apply it to any view it's in